docs/intro.rst

```diff
@@ -34,6 +34,21 @@ to the D-Wave system's numerically indexed qubits, a mapping known as :term:`min
 >>> print(dnx.min_vertex_cover(s5, sampler))
 [0]
 
+Using the D-Wave System as a Sampler
+====================================
+
+The :std:doc:`dimod <dimod:index>` API makes it possible to easily interchange samplers
+in your code. For example, you might develop code using :std:doc:`dwave_neal <neal:index>`,
+Ocean's classical simulated annealing sampler, and then swap in a D-Wave system
```

Comment:
I feel like "classical simulated annealing" is a bit redundant :smile:

Reply:
Deleted